charts/docker-hub-rate-limit-exporter-chart/values.yaml

```diff
@@ -33,29 +60,30 @@ serviceAccount:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name: ""
-
-# Labels to add to each pod.
-podLabels: {}
-    # myPodLabel: labelName
+    imagePullSecrets: []
 
 podAnnotations: {}
 
-podSecurityContext:
-    {}
-    # fsGroup: 2000
-
 securityContext:
-    {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  fsGroup: 65534
+  runAsGroup: 65534
+  runAsNonRoot: true
+  runAsUser: 65534
+
+containerSecurityContext: {}
+  # capabilities:
+  #   add:
+  #   - SYS_TIME
 
 service:
-    type: ClusterIP
-    port: 80
+  type: ClusterIP
+  port: 9100
+  targetPort: 9100
+  nodePort:
+  portName: metrics
+  timeoutSeconds: 5
+  annotations:
+    prometheus.io/scrape: "true"
```

Comment:
Whats the merit of this change? Is it because `serviceMonitor.enabled` defaults to false and you want prometheus to automatically scrap metrics in the default config?

Maybe this could be problematic if you have mulitple prometheus instances? Not sure though.